### DISCUSSION TITLE
TV Series marked as Partially Available

### DISCUSSION THREAD

**Discussion by TechTidalTraveler** [sct/overseerr#3335] [Feature Requests]
It would be nice if a TV Series was considered "Available" if all aired episodes have been downloaded and are available.  Currently, a TV series that has future unaired episodes will show as "Partially Available".  I like to filter requests down to the ones that are unavailable, and the way things currently work these types of TV Series will be included.

<img width="796" alt="image" src="https://user-images.githubusercontent.com/10390918/218337864-5ecd1f32-3740-4199-baa9-5cb816e8de93.png">

**Comment by billycao** [comment]
Agree with this.

Adding to this, it would also be nice if a show was marked "Available" if not all seasons were requested.

Seems silly to say season 1 is "Partially Available" if I only requested season 1. That should be reserved for when an issue arises (ie. requested episodes are missing).

As it is it's impossible to tell if "Partially Available" means "something is wrong" or "you didn't want everything/not all of it has aired yet".

**Comment by muffinman0** [comment]
I would also like something like this. Ideally setup so it won't spam notifications as every new episode airs.

**Comment by traiannicolau** [comment]
would be nice to also have the "available" green box inside the season as well to know why it's partially available because right now you don't. you must check and see if the rest of the episodes were supposed to be aired or not, or sonarr just couldn't find an episode.

**Comment by Lockszmith-GH** [comment]
Adding my voice here.
Agree about the confusion.

But this request needs to also specify the conditions that should be mentioned:

Per season - requested
Per season - partially available
Per season - available + monitoring
Per season - available + complete

Per show - requested
Per Show - partially available
Per Show - available + ongoing
Per Show - requests available (aka partially available, but all requests fulfilled)

If anyone can think of an additional state, please reply.

Maybe the solution is to change terms so that there is a unique combination of words to describe all states.

**Comment by drohack** [comment]
+1 to this. It's annoying to look at the Requests tab and have a bunch of shows that are "Partially Available", but are up to date on what episodes have been grabbed.

**Comment by TinFoilFox** [comment]
I also agree. I would like to see this feature. I came to make this same suggestion only to see someone else had already made it. 👍

**Comment by Link440** [comment]
Honestly +1 because I sort my requests by Available or not and it's full of TV shows where I was asked for only one season or series currently being broadcast

**Comment by toddke** [comment]
Perhaps it would be sufficient to add another filter option -- "Partially Available"? Then, remove any "partially available" items when filtering on "Unavailable".
